modules/network/smb2man/libsmb2/lib/pdu.c

```
@@ -0,0 +1,1164 @@
+int
+smb2_process_payload_variable(struct smb2_context *smb2, struct smb2_pdu *pdu)
+{
+        if (smb2_is_server(smb2)) {
+                return smb2_process_request_payload_variable(smb2, pdu);
+        }
+        else {
+                return smb2_process_reply_payload_variable(smb2, pdu);
+        }
+}
+
+void smb2_timeout_pdus(struct smb2_context *smb2)
```

Cppcheck (2.21.0)

[style] The function 'smb2_timeout_pdus' is never used.

(unusedFunction)